Src/WitsmlExplorer.Frontend/components/ContentViews/JobsView.tsx

```diff
@@ -125,9 +138,33 @@ export const JobsView = (): React.ReactElement => {
     { property: "username", label: "Ordered by", type: ContentType.String }
   ];
 
-  const cancelJob = async (jobId: string) => {
+  const cancel = async (jobId: string) => {
     dispatchOperation({ type: OperationType.HideContextMenu });
-    JobService.cancelJob(jobId);
+    dispatchOperation({ type: OperationType.HideModal });
+    const currentlyCancellingJobs = cancellingJobs;
+    currentlyCancellingJobs.push(jobId);
+    setCancellingJobs(currentlyCancellingJobs);
+    await JobService.cancelJob(jobId);
+  };
+
+  const onClickCancel = async (jobId: string) => {
+    const confirmation = (
+      <ConfirmModal
+        heading={"Cancel job?"}
+        content={<span>Do you like to cancel running job?</span>}
```

Comment:
Also change the heading and content here, similar to my other comment. 

As we also need to add this to the context menu option, maybe move it to a common component or function that can be reused all 3 locations?